README.rst

```diff
@@ -280,6 +274,17 @@ Here is an example of fetch a row:
     }
     $dbh = null;
     echo "OK\n";
+  $>
+
+Limitations in the Driver
```

Comment:
This section already exists -- I just moved it to the end of the "Using" section.